airflow/contrib/operators/mongo_to_s3.py

```diff
@@ -105,7 +106,8 @@ def _stringify(self, iterable, joinable='\n'):
             [json.dumps(doc, default=json_util.default) for doc in iterable]
         )
 
-    def transform(self, docs):
+    @staticmethod
```

Comment:
Nit, I always put static function above the non-static one in the class order

Reply:
+1, I would also prefer static methods above instance methods